caravel/viz.py

```diff
@@ -591,6 +591,51 @@ def get_data(self):
         }
 
 
+class BigNumberTotalViz(BaseViz):
+
+    """Put emphasis on a single metric with this big number viz"""
+
+    viz_type = "big_number_total"
+    verbose_name = "Big Number Total"
```

Comment:
Maybe this new viz should be just "Big Number" and the previous one should say "Big Number with Trendline" or something. The `viz_type` we cannot change because of backwards compatibility, but the verbose_name we can change all we want.